modules/Microsoft.Compute/virtualMachines/deploy.bicep

```diff
@@ -60,7 +60,7 @@ param certificatesToBeInstalled array = []
   'Low'
   'Spot'
 ])
-param vmPriority string = 'Regular'
+param priority string = 'Regular'
```

Comment:
General: do massive input param updates deserve a minor version bump? That would apply also to the previous PR

Reply:
It should be major, but as said, since we're not at the 1.0 repo release there's no point to update it. My understanding was we're updating minor at least for breaking changes like this one. So, is it now a given that we're never updating minor either? Is that something up for discussion sooner or later? In case, we're missing an issue.

Reply:
Maybe I just missed the memo. Let's bring it up in our next dicussion session @eriqua :)

Reply:
Hey @eriqua, I opened the issue #2808